docs/go.mod

```diff
@@ -0,0 +1,22 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
```

Comment:
Afaik, this file should be maintained by Hugo. Dose it make sense to add comment or reference to the related doc in Flink repo[1] to let user know how to update the file for adding new external connectors? Developers who didn't read the doc in Flink might start to edit this file manually which might not be the most correct way we expected. 

[1]https://github.com/apache/flink/blob/c0fa17741c2b3ec643563658e7c0a286fa5c81a1/docs/README.md  (4. In the Flink repository, run the following command to add this newly create module:...)

Reply:
A user needs to reach out to the PMC first to get a new repository created anyway, so I don't think that would be a big problem. But we should collect all these steps into a small runbook on "how to create a new connector" which includes explanation on this step imho

Reply:
A user needs to reach out to the PMC first to get a new repository created anyway, so I don't think that would be a big problem. But we should collect all these steps into a small runbook on "how to create a new connector" which includes explanation on this step imho